src/fides/config/database_settings.py

```diff
@@ -121,6 +130,14 @@ class DatabaseSettings(FidesSettings):
         exclude=True,
     )
 
+    # @field_validator("read_sever", mode="before")
+    # def copy_server_if_no_read_server(
+    #     cls, value: Optional[str], info: ValidationInfo
+    # ) -> str:
+    #     if not value:
+    #         return info.data.get("server")
+    #     return value
+
```

Comment:
drive by comment: do we not need some fallback logic like this? i may be missing it, but it does feel like we want to ensure that things still work just fine if you don't set the `read_server`, i.e. it should fallback to what the `server` value is

Reply:
Implemented this how Neville suggested instead.